WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductFromImage/AddProductFromImageScannedTextView.swift

```diff
@@ -0,0 +1,40 @@
+import SwiftUI
+
+/// Selectable and editable scanned text for generating product details in the "add product from image" flow in `AddProductFromImageView`.
+struct AddProductFromImageScannedTextView: View {
+    @ObservedObject private var viewModel: AddProductFromImageViewModel.ScannedTextViewModel
+
+    init(viewModel: AddProductFromImageViewModel.ScannedTextViewModel) {
+        self.viewModel = viewModel
+    }
+
+    var body: some View {
+        HStack {
+            TextField("", text: $viewModel.text)
```

Comment:
💡 We can set `axis: .vertical` here for iOS 16 and above to make the text field grow vertically and make it easier to read.

Reply:
This text field UI will be updated in the next PR, example screencast pe5sF9-1Es-p2#comment-2495